Packs/EDL/Integrations/EDL/README.md

```diff
@@ -27,36 +27,44 @@ Unlike `PAN-OS EDL Management`, this integration hosts the EDL on the Cortex XSO
+| IP Collapsing                      | For use with PAN-OS (text) format - collapse method for IPs (none, range, CIDR).                                                                                                                                                                     | False        |
+| PAN-OS: drop invalid URL entries   | For use with PAN-OS (text) format - if selected, any URL entry that is not compliant with PAN-OS URL format is dropped instead of being rewritten.                                                                                                   | False        |
+| McAfee Gateway: Indicator List Type | For use with McAfee Web Gateway format - set the indicator list type.                                                                                                                                                                                | False        |
+| Symantec ProxySG: Default Category | For use with Symantec ProxySG format - set the default category for the output.                                                                                                                                                                      | False        |
+| Symantec ProxySG: Listed Categories | For use with Symantec ProxySG format - set the categories that should be listed in the output. If not set, will list all existing categories.                                                                                                        | False        |
+| Show CSV formats as Text           | If selected, CSV format will appear in a textual webpage instead of initiating a file download.                                                                                                                                                      | False        |
+| XSOAR Indicator Page Size          | Internal page size used when querying Cortex XSOAR for the indicators.                                                                                                                                                                               | False        |
+| Maximum CIDR prefix size   | CIDRs with prefix size smaller than this value will not be included. Default is 8.                                                                                                                                                                   | True         |
+| Disable insertion top level domains        | Check if disable top Level Domain in the list. Default is False.                                                                                                                                                                                     | True         |
```

Comment:
Exclude top level domainGlobs | Option to remove top level domainGlobs from the list. For example - *.com. | False